bundles/redhat8.6/packages.txt.gotmpl

```diff
@@ -31,7 +31,7 @@ elfutils-libelf-devel
 libseccomp
 nfs-utils
 iproute-tc
-kernel-headers
-kernel-devel
+kernel-headers-4.18.0-372.103.1.el8_6
+kernel-devel-4.18.0-372.103.1.el8_6
```

Comment:
:/ it always tries to download the 8.8 version

https://github.com/mesosphere/konvoy-image-builder/actions/runs/10254800579/job/28370433396?pr=1133#step:9:2136

Reply:
thats a link to a rhel 8.8 run. it seems like we have to pin it in redhat 8.8 as well.

Reply:
🤦‍♂️ ok at least that explains it, I was starting to look why we would see this issue in 8.6 but not 8.8 now that 8.10 is out.

Reply:
Before pinning, can you please try `unregistering` as described in https://access.redhat.com/solutions/238533

I wonder if `subscription-manager register` automatically subscribes to these repos.
```
subscription-manager repos --disable=rhel-8-for-x86_64-baseos-rpms --disable=rhel-8-for-x86_64-appstream-rpms
```

Reply:
I didn't see any mention of `unregister` in the above article.

Did you mean the part that talks about `subscription-manager release --unset`?

Reply:
I added the disable line, and i think it gets us closer to a package bundle specifically for a EUS point version eg 8.8 but it doesn't necessarily solve the issue of matching kernel headers versions to the kernel.

https://github.com/mesosphere/konvoy-image-builder/pull/1136

should solve disabling it for users who don't need it and some new docs tracked by https://jira.nutanix.com/browse/NCN-101854 can explain what to put for the version and how to get it.